ansible/roles/ocp4-workload-quarkus-workshop/tasks/workload.yml

```diff
@@ -15,6 +15,17 @@
   debug:
     msg: "selected modules list: {{ modules }}"
 
+- name: Create CatalogSource Index 4.6
+  k8s:
+    state: present
+    merge_type:
+    - strategic-merge
+    - merge
+    definition: "{{ lookup('file', item ) | from_yaml }}"
+  loop:
+  - ./files/redhat-operators-index.yaml
+  - ./files/community-operators-index.yaml
```

Comment:
No need to have `./files/` with `lookup('file', ... )`. It by default looks there.

Reply:
Should it Ok with?
  loop:
  - redhat-operators-index.yaml
  - community-operators-index.yaml

Reply:
Updated